ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.ts

```diff
@@ -132,12 +122,16 @@ export class ArchivesLoggingFiltersComponent implements OnInit, OnChanges, OnDes
 
     dateTimeFilterChange = new Subject();
 
-    publishMinDate: {year: number; month: number; day: number} = null;
-    publishMaxDate: {year: number; month: number; day: number} = null;
-    activeMinDate: {year: number; month: number; day: number} = null;
-    activeMaxDate: {year: number; month: number; day: number} = null;
+    publishMinDate = null;
+    publishMaxDate = null;
+    activeMinDate = null;
+    activeMaxDate = null;
```

Comment:
I think you do not need anymore these four variables as the range is fully managed by the date range picker component